### ISSUE TITLE
Some CSV Exports not working properly

### ISSUE THREAD

**Issue by Aaronsss** [RotorHazard/RotorHazard#548]
3.0.0 beta 1 has worked solidly for the day but trying to export the times when I got home has been an issue.

Every export option works properly except for:
CSV (Frienly) / All
CSV (Friendly) / Results

Trying to export from the latest version of Firefox desktop. 

I have tried with multiple database files (we generated 5 throughout the day) none of them appear to work

**Comment by Aaronsss** [comment]
error from the server log:

```
T2752: Built results data in: 0.211102s
Exporting data via csv_all
Exception via catchLogExceptionsWrapper
Traceback (most recent call last):
  File "/home/pi/RotorHazard/src/server/RHUtils.py", line 165, in wrapper
    return func(*args, **kwargs)
  File "server.py", line 1868, in on_export_database
    export_result = export_manager.export(exporter, Database, PageCache)
  File "/home/pi/RotorHazard/src/server/data_export.py", line 51, in export
    return self.exporters[exporter_id].export(Database, PageCache)
  File "/home/pi/RotorHazard/src/server/data_export.py", line 61, in export
    data = self.assembler(Database, PageCache)
  File "/home/pi/RotorHazard/src/server/data_export_csv.py", line 29, in assemble_all
    payload['Results'] = assemble_results(Database, PageCache)
  File "/home/pi/RotorHazard/src/server/data_export_csv.py", line 224, in assemble_results
    if race_class:
UnboundLocalError: local variable 'race_class' referenced before assignment

```

**Comment by HazardCreative** [comment]
Can you please supply one of your database files and log with the issue?

**Comment by Aaronsss** [comment]
Requested files attached. I had to change the .db file to .txt so github would let me add it

[csv creation log.txt](https://github.com/RotorHazard/RotorHazard/files/6258262/csv.creation.log.txt)

[database_20210404_145846.txt](https://github.com/RotorHazard/RotorHazard/files/6258237/database_20210404_145846.txt)

**Comment by HazardCreative** [comment]
Thanks! I'll use this to dig further.

For future reference, you can go to ... > View Server Log and use the "Download Logs" button there. It'll give you a zip file with database and log files included. GitHub accepts these zips.

**Comment by HazardCreative** [comment]
Fixed via 04d0075 .
